### PR TITLE
First configuration of files

### PR DESCRIPTION
**Pull request by alex1779**
Added files README.md, MIT.md, .gitignore. Added files index.html, css folder.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (alex1779) [inline index.html:23]: Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <input type="submit" id="submit">
+
+      </form>
+    </main>
+
+  </body>
+
+
+
+  </body>
```
Comment (alex1779) [inline index.html:62]: It looks like the body tag is duplicated. Please fix that. As additional comment, try not to let blank lines if do not be necessary.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    <header>
+      <h1 id="title"><title>Survey Form</title>Survey Form</h1>
+      <p id="description" > "What do you thing about our services"</p>
+    </header>
+    <main>
+      <form id="survey-form">
+l       <label id="name-label" for="name" >Please enter your name:</label>
+        <input id="name" type="text" required placeholder="Alejandro Maggioni">
+
```
Comment (alex1779) [inline index.html:34]: Could you check line 32 does not contain unwanted characters.

**Review by alex1779 — COMMENTED** [review]
Hi @alex1779, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good commit messages✔️
No linter errors ✔️
Design looks good✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — APPROVED** [review]
Hi @alex1779, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- Good job for using the right GitHub Flow 💯
- Review comments are descriptive and respectful 👍

## Optional suggestions
**[Optional] It's recommended to use some resource links in your reviews, so that the code author can study about the reason.** 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**